pwiz_tools/Skyline/Controls/Graphs/DetectionsGraphController.cs

```diff
@@ -153,7 +153,7 @@ public class Settings
             public static float QValueCutoff
             {
                 get => Properties.Settings.Default.DetectionsQValueCutoff;
-                set => Properties.Settings.Default.DetectionsQValueCutoff = value;
+                set => Properties.Settings.Default.DetectionsQValueCutoff = value; 

```

Comment:
Probably not worth making this line-ending change, since it is the only change to this file.